src/devices/cpu/rsp/rsp_dasm.cpp

```diff
@@ -341,12 +341,11 @@ offs_t rsp_dasm_one(std::ostream &stream, offs_t pc, uint32_t op)
 	return 4 | flags | DASMFLAG_SUPPORTED;
 }
 
-offs_t rsp_dasm_one(char *buffer, offs_t pc, uint32_t op)
```

Comment:
As with PPC, this trampoline isn't really very useful - the code without it isn't any more ugly.